2022-03-25 15:28:03,513 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2022-03-25 15:28:03,514 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Users\balin\AppData\Local\Programs\Python\Python39\lib\site-packages\instabot\bot\bot.py
2022-03-25 15:28:03,515 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2022-03-25 15:28:03,960 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2022-03-25 15:28:03,960 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2022-03-25 15:28:03,960 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2022-03-25 15:28:03,961 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 5 minutes.
